test/failure_handler/Makefile

```diff
@@ -38,7 +38,7 @@ SOURCES := ${SRC_DIR}/jdk/test/failurehandler/*.java                   \
 
 CONF_DIR = src/share/conf
 
-JAVA_RELEASE = 7
```

Comment:
could you please update `DEPENDENCES` section in `test/failure_handler/README`?

Reply:
done